src/Workspaces/Core/Portable/Notification/GlobalOperationNotificationService.cs

```diff
@@ -20,9 +21,17 @@ internal class GlobalOperationNotificationService : AbstractGlobalOperationNotif
         private readonly SimpleTaskQueue _eventQueue = new SimpleTaskQueue(TaskScheduler.Default);
         private readonly EventMap _eventMap = new EventMap();
 
+        private readonly IAsynchronousOperationListener _listener;
+
+        [Obsolete("Pass an explicit " + nameof(IAsynchronousOperationListener) + " when constructing this service.")]
         public GlobalOperationNotificationService()
```

Comment:
This is an internal class. Can we just delete it and see if it breaks anyone?

Reply:
If this is for dev15.7.x, probably not. If it's for master, then sure.

Reply:
Agreed. This is targeting master though and I can't imagine taking it for dev15.7.x since it's just an opportunistic refactoring.

Reply:
➡️ I removed the obsolete constructor from the commit.